test/browser_tests/spec_suites/organisms/pagination.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var Blog = require(
+    '../../page_objects/page_blog.js'
+  );
+
+describe( 'Pagination', function() {
+  var page;
+
+  beforeEach( function() {
+    page = new Blog();
+    page.get();
+  } );
+
+  it( 'should navigate to the second page', function() {
+    page.paginationNextBtn.click()
+    browser.sleep( 1000 );
```

Comment:
Is there an alternative to using sleep here? https://github.com/angular/protractor/issues/685